dashboard/src/pages/TreeDetails/Tabs/TestCards.tsx

```diff
@@ -158,14 +158,23 @@ const StatusChart = ({ statusCounts, title }: IStatusChart): JSX.Element => {
     success: 0,
     fail: 0,
     skip: 0,
+    error: 0,
+    done: 0,
+    miss: 0,
   };
 
   const statusKeys = Object.keys(statusCounts) as Status[];
   statusKeys.forEach(status => {
-    if (errorStatusSet.has(status as ErrorStatus)) {
+    if (status === 'FAIL') {
       groupedStatus.fail += statusCounts[status] ?? 0;
     } else if (status === 'SKIP') {
       groupedStatus.skip += 1;
+    } else if (status === 'ERROR') {
+      groupedStatus.error += 1;
+    } else if (status === 'DONE') {
+      groupedStatus.done += 1;
+    } else if (status === 'MISS') {
+      groupedStatus.miss += 1;
     } else {
       groupedStatus.success += statusCounts[status] ?? 0;
     }
```

Comment:
We don't need this foreach anymore, we are not grouping the status that comes from the backend, this can come formated straight from the backend